docs/getting-started/continuous_integration.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: default
+title: Continuous Integration
+parent: Getting started
+nav_order: 5
+permalink: /getting-started/continuous-integration/
+---
+
+# Continuous Integration
+
+OSS-Fuzz offers **CIFuzz**, which will run your fuzz targets each time a pull request
+is submitted, for projects hosted on GitHub. This allows you to detect and
+fix bugs before they are submitted.
```

Comment:
"submitted" isn't super clear let's say "before they make it into your codebase"

Reply:
Fixed.